### PR TITLE
Update Makefile

### PR DESCRIPTION
**Pull request by soulshake**
Some ideas to try

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -11,23 +11,22 @@ help:
...
 builds:
 	@echo "Building Docker Images... 🐳"
-	docker-compose build --no-cache
+	docker-compose build
 	@echo "Done! ✨"
-up:
+up: builds
 	@echo "Starting library... ⚡️ "
-	docker-compose up --build
+	docker-compose up --build -d
```
Comment (soulshake) [inline Makefile:18]: I haven't tried running this but without the `-d` to send to the background, I would expect this line to block things from proceeding further.

File: Makefile [hunk 2]
```diff
@@ -11,23 +11,22 @@ help:
...
+	@docker-compose logs --follow
 stop:
 	@echo "Stopping Library 🛑"
 	docker-compose down
 	@echo "Done! ✨"
 	@echo "Sucessfully stopped Library"
-all:
-	@echo "Making library..."
-	make builds && make up
+all: builds up
```
Comment (soulshake) [inline Makefile:29]: Adding `builds` and `up` as prerequisites will tell `make` to fulfill those targets beforehand, so you don't need to run `make builds && make up` within the rule.

File: Makefile [hunk 3]
```diff
@@ -11,23 +11,22 @@ help:
 
 builds:
 	@echo "Building Docker Images... 🐳"
-	docker-compose build --no-cache
+	docker-compose build
```
Comment (soulshake) [inline Makefile:14]: Do you just enjoy pain, or....? :sweat_smile:

File: Makefile (hunk 2 above)
Comment (SammyRobensParadise) [reply Makefile:29]: wow thats super helpful! thanks! :)

File: Makefile (hunk 3 above)
Comment (SammyRobensParadise) [reply Makefile:14]: hahaha, good point!

**Pull request merged**